legal_db/tests.py

```diff
@@ -1,3 +1,69 @@
-from django.test import TestCase  # noqa
+from django.test import TestCase
+from django.urls import reverse
 
-# Create your tests here.
+from .models import Link, Scholarship
+
+
+def create_scholarship():
```

Comment:
I recommend using something like Factory Boy (https://github.com/FactoryBoy/factory_boy) to make this type of pattern easier.